nginx.conf

```diff
@@ -0,0 +1,21 @@
+worker_processes  1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    server {
+        root /usr/share/nginx/html;
+
+        gzip on;
+        gzip_types text/plain application/xml;
```

Comment:
Should `application/json` be included for cases when you're sending JSON from the frontend to the backend?